server/src/assignment/assignment.service.ts

```diff
@@ -228,6 +239,14 @@ function computeStatusTab(project, lupteam) {
     return 'archive';
   }
 
+  if (
+    project.dcp_publicstatus === 717170005 // Prefiled
+      && project.dcp_ulurp_nonulurp === 717170001 // ULURP
+      && project.dcp_validatedcommunitydistricts.includes(recodedCbFullName)) { // TODO
+    console.log('Prefiled!');
+    return 'upcoming';
+  }
```

Comment:
The data for this is getting filtered out beforehand at the web api moment

Reply:
Seems to have something to do with lupteammember doesn't have a team member assignment because they won't have that record

Reply:
prefiled projects just aren't being included for some rason